ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -591,19 +603,47 @@ ORK_CLASS_AVAILABLE
 + (instancetype)choiceWithText:(NSString *)text value:(id<NSCopying, NSCoding, NSObject>)value;
 
 /**
+ Returns a choice object that includes the specified primary text and exclusivity.
+ 
+ @param text        The primary text that describes the choice in a localized string.
+ @param value       The value to record in a result object when this item is selected.
+ @param exclusive   Whether this choice is to be considered exclusive within the set of choices.
+ 
+ @return A text choice instance.
+ */
++ (instancetype)choiceWithText:(NSString *)text value:(id<NSCopying, NSCoding, NSObject>)value exclusive:(BOOL)exclusive;
+
+/**
  Returns an initialized text choice object using the specified primary and detail text.
  
+ This method is a convenience initializer.
+ 
+ @param text        The primary text that describes the choice in a localized string.
+ @param detailText  The detail text to display below the primary text, in a localized string.
+ @param value       The value to record in a result object when this item is selected.
+ 
+ @return An initialized text choice.
+ */
+- (instancetype)initWithText:(NSString *)text
+                  detailText:(nullable NSString *)detailText
+                       value:(id<NSCopying, NSCoding, NSObject>)value;
+
+/**
+ Returns an initialized text choice object using the specified primary text, detail text, and exclusivity.
+ 
  This method is the designated initializer.
  
  @param text        The primary text that describes the choice in a localized string.
  @param detailText  The detail text to display below the primary text, in a localized string.
  @param value       The value to record in a result object when this item is selected.
+ @param exclusive   Whether this choice is to be considered exclusive within the set of choices.
  
  @return An initialized text choice.
  */
 - (instancetype)initWithText:(NSString *)text
```

Comment:
Ok.  Yeah I was trying to prevent a breaking API change.  So you want
```
- (instancetype)initWithText:(NSString *)text
                  detailText:(nullable NSString *)detailText
                       value:(id<NSCopying, NSCoding, NSObject>)value;
```
removed.  What about the choiceWithText methods?  There were already two of them, with one not forcing you to pass the detailText.  I followed that example, and added two more signatures, allowing you to pass in exclusive.  Do you want those retained?